dashboard/README.md

```diff
@@ -1,70 +1,198 @@
-# Getting Started with Create React App
+## Foreword
+
+Pbench Dashboard is a web-based platform for consuming indexed performance benchmark data. The platform provides a consolidated view of benchmark data within tables, charts, and other powerful visualizations. Users are able to quickly navigate through benchmark data and tune analytics through comparison tools for in-depth analysis.
+
+## Scaffolding
+
+```bash
```

Comment:
This is nice, but how easy is this tree going to be to maintain? It's going to change a lot, pretty much with each PR, at least until we have a "fully functional" dashboard.

I wonder if it might be better to have nested H2/H3 headers giving a generic description of the directory tree and the sort of files we put there? I think the description of each directory is more useful than just a list (without explanation) of the files we happen to have in that directory at this moment.

E.g., "public" has project-wide static web files, "src" contains the dashboard Javascript and LESS/CSS sources, with "actions" and "assets" and "modules" as subdirectories, etc.

Reply:
If people want a file list, they'll look at the GitHub "code" link (or just use the `ls` command...) instead of the `README.md`.  We should have things in here that the reader would have difficulty getting any other way (like, as Dave says, what the directories are _for_, rather than what they are).

Reply:
Pruned the tree structure to 2 level and provided simple description for each directory

Reply:
Most of these don't require long or elaborate descriptions, but for many I think more than a few words in a comment would be useful.

Again, I think that the structured tree format is "cute" and it's great at showing the *hierarchy* but not the best way to describe the purpose of each directory. Unfortunately I don't think GitHub Markdown is rich enough to combine the two styles (for example, making each name in your tree "graphic" an active `[link](#to a sub-section)` with a description, which I think might be an ideal representation).

Without that, though, I think simple nested subsections might work better overall.

```
# Dashboard directory structure

## public

Contains the root application `index.html` and React template artifacts.

(BTW, do we really want/need to retain the React logos here? We don't use them, do
we?)

## server

The source for an NPM express server that's used in local developer mode to reflect
the server `/api/v1/endpoints` API call to a remote server. This enables local
debugging against a real Pbench Server.

## src

The Pbench Dashboard Javascript source plus additional CSS/LESS and artifacts.

### assets

[...]
```

The `# Assets` section below probably belongs in this section, or possibly even the `#### images` sub-section, since the text really explains the distinction between the top-level `public` assets and the `src/assets/images` assets.

Reply:
Updated the file structure